pkg/columns/README.md

```diff
@@ -0,0 +1,102 @@
+# Columns Library
+
```

Comment:
This package documentation could be in `pkg/columns/doc.go` and `pkg/columns/example_test.go`, so that go doc can pick it up.

Example:
* Package documentation: https://pkg.go.dev/encoding/gob
* Source `doc.go`: https://go.dev/src/encoding/gob/doc.go?m=text
* Source `example_{,*_}test.go`: https://go.dev/src/encoding/gob/example_test.go?m=text

Reply:
Good point! I moved most of the documentation and the most important examples now. 👍